src/middlewares/authenticator.js

```diff
@@ -19,33 +19,35 @@ class Authenticator {
   }
 
   /**
-   * @method verifyToken
+   * @method authenticateUser
    * @description Verifies that user is authenticated
    * @param {object} req - The Request Object
    * @param {object} res - The Response Object
    * @param {object} next - The next function
    * @returns {object} - JSON response object
    */
-  static async verifyToken(req, res, next) {
+  static async authenticateUser(req, res, next) {
     let token = '';
     /* eslint-disable no-unused-vars */
     let str = '';
-    if (req.get('Authorization').startsWith('Bearer')) {
+    const authorizationHeader = req.get('Authorization');
+    if (!authorizationHeader) return res.status(400).json(responseFormat({ status: 'error', message: 'No authorization token present in header', }));
+    if (authorizationHeader.startsWith('Bearer')) {
       [str, token] = req.get('Authorization').split(' ');
     } else {
       token = req.get('Authorization') ? req.get('Authorization') : req.headers.token;
```

Comment:
Let us be consistent with where our token should be.

Reply:
@Temmyogunbo The token field is consistent, what we are checking is the manner in which it is included, so we can refactor it.